src/js/providers/html5.js

```diff
@@ -428,6 +428,15 @@ define([
             _setLevels(item.sources);
             this.sendMediaType(item.sources);
 
+            // canSeek to true to prevent canPlayHandler from starting the video
+            _canSeek = true;
```

Comment:
Good comment, it helped me understand your goal.

I think we can solve the problem a different way, since _canSeek is technically false (because we aren't ready to seek yet). You identified that the issue is in _canPlayHandler. Maybe if we don't call _canPlayHandler inside of _onLoadedMetaData that would suffice for preventing the video starting.